packages/terra-file-path/.npmignore

```diff
@@ -0,0 +1,5 @@
+.eslintrc.js
+*.log
+jest.config.js
+jest.enzymeSetup.js
+tests
```

Comment:
nit:
```suggestion
*.log
.eslintrc.js
jest.config.js
jest.enzymeSetup.js
tests
```

Reply:
I did a sort alphabetically and this was the order, so I will keep it like this

Reply:
Ah okay, makes sense